### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by mpjlu**
add ml workloads readme introduction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,11 +62,35 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
 
 3. Logistic Regression (lr)
 
-    This workload benchmarks Logistic Regression implemented in Spark-MLLib examples. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
+    This workload benchmarks Logistic Regression (lr) implemented in Spark-MLLib. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
```
Comment (VinceShieh) [inline README.md]: This workload benchmarks Logistic Regression with LBFGS optimizer (based on RDD-API).

File: README.md [hunk 2]
```diff
@@ -62,11 +62,35 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
...
-    This workload benchmarks Alternating Least Squares implememnted in Spark-MLLib examples. The input data set is generated by RatingDataGenerator for a product recommendation system.
+    This workload benchmarks Alternating Least Squares implememnted in Spark-MLLib. The input data set is generated by RatingDataGenerator for a product recommendation system.
+
+5. Gradient Boosting Tree (gbt)
+
+    This workload benchmarks Gradient Boosting Tree implememnted in Spark-MLLib. The input data set is generated by GradientBoostingTreeDataGenerator.
+
+6. Linear Regression (lir)
+
+    This workload benchmarks Linear Regression (lir) implemented in Spark-MLLib. Linear Regreesion is realized with SGD. The input data set is generated by LinearRegressionDataGenerator.
```
Comment (VinceShieh) [inline README.md]: This workload benchmarks Logistic Regression with SGD optimizer (based on RDD-API).

File: README.md (hunk 1 above)
Comment (VinceShieh) [inline README.md]: The input data set is generated by LogisticRegressionDataGenerator

File: README.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
 
 1. Bayesian Classification (bayes)
 
-    This workload benchmarks NaiveBayesian Classification implemented in Spark-MLLib/Mahout examples.
+    This workload benchmarks NaiveBayesian Classification implemented in Spark-MLLib/Mahout.
```
Comment (chenghao-intel) [inline README.md]: why do you leave `Mahout` here?

File: README.md [hunk 4]
```diff
@@ -62,11 +62,35 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
...
 
-    This workload benchmarks Logistic Regression implemented in Spark-MLLib examples. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
+    This workload benchmarks Logistic Regression (lr) implemented in Spark-MLLib. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
 
 4. Alternating Least Squares (als)
 
-    This workload benchmarks Alternating Least Squares implememnted in Spark-MLLib examples. The input data set is generated by RatingDataGenerator for a product recommendation system.
+    This workload benchmarks Alternating Least Squares implememnted in Spark-MLLib. The input data set is generated by RatingDataGenerator for a product recommendation system.
+
+5. Gradient Boosting Tree (gbt)
```
Comment (chenghao-intel) [inline README.md]: Upper Case please.

File: README.md [hunk 5]
```diff
@@ -62,11 +62,35 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
...
+
+    This workload benchmarks Gradient Boosting Tree implememnted in Spark-MLLib. The input data set is generated by GradientBoostingTreeDataGenerator.
+
+6. Linear Regression (lir)
```
Comment (chenghao-intel) [inline README.md]: `lir` -> `LIR`

File: README.md (hunk 2 above)
Comment (chenghao-intel) [inline README.md]: same here

File: README.md (hunk 3 above)
Comment (VinceShieh) [reply README.md]: datagen is from mahout

Comment (chenghao-intel) [reply README.md]: thanks for the explanation, but we don't want to confuse the end user right?

Comment (mpjlu) [reply README.md]: ok, I will remove mahout.

File: README.md (hunk 5 above)
Comment (mpjlu) [reply README.md]: ok, I will change all to upper case

**Pull request merged**